src/jit/nodeinfo.h

```diff
@@ -25,12 +25,9 @@ class TreeNodeInfo
         dstCandsIndex          = 0;
         internalCandsIndex     = 0;
         isLocalDefUse          = false;
-        isLsraAdded            = false;
         isDelayFree            = false;
         hasDelayFreeSrc        = false;
         isTgtPref              = false;
-        regOptional            = false;
-        definesAnyRegisters    = false;
```

Comment:
It appears that `definesAnyRegisters` is no longer used but it was not removed.

Reply:
It looks like `loc` is also unused and can be removed

Reply:
Once these are removed TreeNodeInfo contains only `char` members and has 6 bytes. This results in alignment holes in `LocationInfo`:
```
	|struct LocationInfo {
0000	|	TreeNodeInfo info;
0008	|	unsigned int loc;
0010	|	Interval* interval;
0018	|	GenTree* treeNode;
	|};
```
You should probably put the pointers first, followed by `loc` and then `info`.